pkg/model/components/etcdmanager/model.go

```diff
@@ -478,6 +478,7 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster kops.EtcdClusterSpec, instance
 				fmt.Sprintf("%s=%s", openstack.TagClusterName, b.Cluster.Name),
 			}
 			config.VolumeNameTag = openstack.TagNameEtcdClusterPrefix + etcdCluster.Name
+			config.NetworkCIDR = etcdCluster.Manager.NetworkCIDR
```

Comment:
can etcdCluster.Manager be nil?

Reply:
Yes I think so (added a check for it at another place but not here), and there should be a check if the network cidr should even be set.